### ISSUE TITLE
Network Pods

### ISSUE THREAD

**Issue by Ramane19** [kubernetes/kubeadm#1609]
After doing the kubeadm init and setting up the network pod with weave, I get the following error while looking at the pods

@k8s-sig-openstack-test-failures

kubectl get pods --all-namespaces
NAMESPACE     NAME                            READY                STATUS                           RESTARTS   AGE
kube-system   coredns-fb8b8dccf-d9pp8                   0/1     CrashLoopBackOff   6          11m
kube-system   coredns-fb8b8dccf-rkqcj                   0/1     CrashLoopBackOff   6          11m
kube-system   etcd-mola-virtualbox                      1/1     Running            0          10m
kube-system   kube-apiserver-mola-virtualbox            1/1     Running            0          10m
kube-system   kube-controller-manager-mola-virtualbox   1/1     Running            0          10m
kube-system   kube-proxy-t4ld7                          1/1     Running            0          11m
kube-system   kube-scheduler-mola-virtualbox            1/1     Running            0          10m
kube-system   weave-net-kc79d                           2/2     Running            0          8m5s

**Comment by Ramane19** [comment]
@christopherhein

**Comment by neolit123** [comment]
you can try the suggestions in https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm/ about coredns crashloop.

**Comment by platy** [comment]
I'm having this issue too. The troubleshooting section suggests 2 things that could be the problem : 
- an issue between older docker versions and SELinux
- a DNS lookup loop which could be caused by having systemd-resolved on the host

I confirmed that I did have the systemd-resolved issue and fixed using the suggestion to add `resolvConf: /run/systemd/resolve/resolv.conf` to `/var/lib/kubelet/config.yaml`.

I still have the issue and I am using the recommended version of Docker : 18.06.2-ce

Is there a way to get some logs from coredns?
